src/debug/inc/arm/primitives.h

```diff
@@ -30,7 +30,11 @@ typedef DPTR(CORDB_ADDRESS_TYPE)    PTR_CORDB_ADDRESS_TYPE;
 #define STACKWALK_CONTROLPC_ADJUST_OFFSET 2
 
 #define CORDbg_BREAK_INSTRUCTION_SIZE 2
+#ifdef LINUX32
```

Comment:
Why? All changes under `LINUX32` in this PR are Linux-specific, but `PLATFORM_UNIX` could mean macOS or any other unix.